packages/mobile/src/editor/screens/LessonPlanEditor.js

```diff
@@ -1,12 +1,35 @@
 import React from 'react';
-import { SafeAreaView, Text } from 'react-native';
+import LessonPlanHeader from '../../components/LessonPlanHeader.js';
+import { StyleSheet, SafeAreaView, View, ScrollView } from 'react-native';
+import LessonSection from '../../components/LessonSection.js';
+import LessonPlanNotes from '../../components/LessonPlanNotes.js';
+import SaveButton from '../../components/SaveButton.js';
 
-const LessonPlanEditor = ({ navigation }) => {
+const LessonPlanEditor = () => {
   return (
-    <SafeAreaView>
-      <Text>Lesson plan editor here!</Text>
+    <SafeAreaView
+      style={{ flex: 1, backgroundColor: '#FFFAF5', paddingBottom: 25 }}>
+      <View style={{ flex: 2.5 }}>
+        <LessonPlanHeader />
+      </View>
+      <ScrollView style={styles.scrollView}>
+        <View style={{ flexDirection: 'column', alignItems: 'center' }}>
+          <LessonSection subtitle={'Warm Up'} />
+          <LessonSection subtitle={'Main Lesson'} />
+          <LessonSection subtitle={'Cool Down'} />
+          <LessonPlanNotes subtitle={'Notes'} />
+          <SaveButton />
+        </View>
+      </ScrollView>
     </SafeAreaView>
   );
 };
 
+const styles = StyleSheet.create({
+  scrollView: {
+    marginHorizontal: 20,
+    height: 600
```

Comment:
I would avoid setting a hard number here for `height` because screen size can change. Try using percentages.